tests/conftest.py

```diff
@@ -233,22 +247,31 @@ def func(line, assert_exit_code=0, stdin=None, matcher=False):
             raise (
                 Exception(
                     (
-                        "CliTest run_line exit_code assertion failed!\n"
-                        "Line:\n{}\nexited with {} when expecting {}\n"
-                        "stdout:\n{}\nstderr:\n{}\nnetwork calls recorded:"
-                        "\n  {}"
+                        """
+CliTest run_line exit_code assertion failed!
+Line:
+  {}
+exited with {} when expecting {}
+
+stdout:
+{}
+stderr:
+{}
+network calls recorded:
+{}"""
```

Comment:
Similar feedback for this: I recommend an f-string here.

Reply:
This predated the independent PR for this change, where we put together revisions. So if I did my rebase correctly, this change should disappear from the history.